pages/resources.jsx

```diff
@@ -9,6 +11,12 @@ export default function About({ ...props }) {
         <title>PuMP</title>
       </Head>
       <PageLayout>
+        <Title
```

Comment:
Example is on the wrongg page haha can we move this to the uas.jsx file under pages/resources folder. Also can you help me rename the function to Resources instead of About

Reply:
Yep, just tossed it here to test. Will add the titles to their pages according to available designs.

Reply:
Removed in next commit, I'll make another PR putting all the titles on pages